src/openrct2/drawing/Drawing.String.cpp

```diff
@@ -313,7 +313,7 @@ void draw_string_centred_raw(rct_drawpixelinfo* dpi, int32_t x, int32_t y, int32
 {
     ScreenCoordsXY screenCoords(dpi->x, dpi->y);
     gCurrentFontSpriteBase = FONT_SPRITE_BASE_MEDIUM;
-    gfx_draw_string(dpi, (char*)"", COLOUR_BLACK, screenCoords);
+    gfx_draw_string(dpi, const_cast<char*>(""), COLOUR_BLACK, screenCoords);
```

Comment:
I dont think a cast is required, passing ```""``` should be of type ```const char*```

Reply:
Its removing the const I believe. Really all of the draw_strings need to be modified to take const.

Reply:
I looked at it, it already takes ```const_utf8string```